src/controllers/confirmation.controller.ts

```diff
@@ -70,12 +72,21 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   if (isStillRequired) {
     const filingDueOn = apiResponseData.filing_due_on;
 
+    if (apiResponseStatus === 400) {
+      const cannotUseReason: string = eligibilityReasonCode[apiResponseData.reason_code];
```

Comment:
If somehow apiResponseData.reason_code is null. What happens here (Just thinking if the api returns a 400 for some reason other than the eligibility check fails)

Reply:
Spec says that for a 400 response reason_code should always be set, so if it isn't we should probably just show the 'Sorry, service failed' screen I guess?